rules/seo-checklist/rule.md

```diff
@@ -16,8 +16,15 @@ related:
 
 ---
 
+Do you find yourself reworking pages on your website that weren't created with SEO in mind?
+
 One of the key challenges for a business is ensuring that its webpages are favored by search engines. It's a delicate balance between providing value to your audience and ticking the right boxes for search engine algorithms.
-This webpage template is tailored to incorporate best SEO practices. Using this when you create a webpage ensures that it starts with a solid baseline SEO. This can also reduce the future pain of reworking existing pages that weren't created with SEO in mind.
+
+The best way to make sure your website has good SEO is to ensure that every that every webpage you create starts with solid SEO. To do this, you should use an SEO components checklist. The checklist below is tailored to incorporate best SEO practices for a new webpage. If you are creating a webpage, you should make sure your page includes all of the elements on the list.
```

Comment:
change from 
  that every that every
to
  that every

Reply:
done